multicluster/controllers/multicluster/leader_clusterset_controller.go

```diff
@@ -83,7 +83,8 @@ func (r *LeaderClusterSetReconciler) Reconcile(ctx context.Context, req ctrl.Req
 	klog.InfoS("Received ClusterSet add/update", "clusterset", klog.KObj(clusterSet))
 
 	// Handle create or update
-	// If create, make sure the local ClusterClaim is part of the leader config
+	// If create, make sure required local ClusterClaims are defined, the cluster and ClusterSet
+	// IDs are also the same IDs defined in the leader ClusterSet CR.
```

Comment:
"IDs are included in the leader cluster's ClusterSet CR"?

Reply:
Done